Sentry bugs

awisema opened the pull request.

awisema committed:
Fix for HMIS-WAREHOUSE-A9

awisema committed:
Fix for HMIS-WAREHOUSE-A8

eanders commented on app/models/health/claims/view_model.rb:
```diff
@@ -19,73 +18,75 @@ def initialize(patient, sdh_rosters)
 
     protected
 
+    Table = Struct.new('Table', :keys, :patient, :sdh, :variance)
```
How about a comment about what's going on here.  It looks like you're essentially defining a new class:
`Health::Claims::ViewModel::Table` but in an odd way.  Why not?
```
class Table < Struct
end
```
or something similar?

awisema replied:
I replaced this with an OpenStruct -- while they are a little slower, the purpose here is just passing a structured object for downstream readability.

awisema committed:
Replace Table Struct with an OpenStruct.
While Struct is faster, the overhead of creating a class obscures the goal of passing a structured object.
